src/follower.py

```diff
@@ -98,17 +100,26 @@ def convert_camera_angle(self):
         else:
             self._turn_angle = 90
 
-
     """
     Resets the camera to the default position.
 
     The default position is tilted to 120 degrees and panned to 90 degrees.
     """
+    
     def reset_camera(self):
         self._camera.turn_down(120)
         self._camera.pan_servo.write(90)
 
+    """
+    Definitions for turning the camera left and right
+    """
 
+    def _turn_camera_left(self, angle):
```

Comment:
Can we also get rid of the _ in the function name here? This one is (I think) in PEP8, but for consistency (for now), let's get rid of it. We'll go back later and privatize everything.

Reply:
got rid of _ in function name, will need to go through later and privatize for consistency.